src/components/Button/Button.st.css

```diff
@@ -189,3 +189,58 @@
     padding: 10px 16px;
     font: fallback(font(value(MainTextFont)), font(value(LargeFont)));
 }
+
+:vars{
+    secondaryBorderWidth: fallback(value(MainBorderWidth), 1px);
+}
+
+/*upgrage*/
+.root:upgrade{
+    box-sizing: border-box;
+    padding-left: 16px;
+    padding-right: 16px;
+}
+
+/*upgrade tiny*/
+.root:priority(secondary):upgrade:size(tiny), .root:priority(basicSecondary):upgrade:size(tiny){
+    padding-top: "calculate(-, 6px, value(secondaryBorderWidth))";
```

Comment:
Ok, so I found the problem.
First, you should remove the string from all the `calculate` experssions.
i.e. write: 
```padding-top: calculate(...)```
In addition, it seems there's a problem with 
```PrimaryBorderWidth: fallback(value(MainBorderWidth), 0);```
It should be:
```PrimaryBorderWidth: fallback(value(MainBorderWidth), 0px);```
(Notice the `px` at the end)